src/Control/RSS/RSSFeed_Entry.php

```diff
@@ -58,7 +58,7 @@ public function __construct($entry, $titleField, $descriptionField, $authorField
     /**
      * Get the description of this entry
      *
-     * @return DBField Returns the description of the entry.
+     * @return DBField|null Returns the description of the entry.
      */
     public function Title()
```

Comment:
This (and the below) call `obj()` either directly or indirectly, which can return null.